### PR TITLE
[BIOMAGE-1566] removed breadcrumbs and spinning loader

### PR DESCRIPTION
**Pull request by kafkasl**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-1566

#### Link to staging deployment URL 

https://ui-pol-header-ui-605.scp-staging.biomage.net/

#### Somethine else 

This changes will be followed up by a refactor making data processing, exploration, and plots and tables use a single component wrapper.

# Changes
### Code changes
* Removed breadcrumb from headers
* Removed spinning loaders

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Code updates
Have best practices and ongoing refactors being observed in this PR

- [ ] Migrated any selector / reducer used to the new format
- [ ] Validated that current unit tests work as expected and are enough

### Testing
- [x] Unit tests written
- [x] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [x] Deployed to staging
- [ ] Passed integration tests 

To set up easy local testing with inframock, follow the instructions here: https://github.com/hms-dbmi-cellenics/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/hms-dbmi-cellenics/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [x] Relevant Github READMEs updated
- [x] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [x] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/hms-dbmi-cellenics/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by aerlaut** [comment]
Good work with the plot header refactor! It makes things really clear. Btw, when I checked out the plot, the header contains duplicated buttons for feedback and invite. There should only be one. Can you please fix this? 

image

**Review by vickymorrison — APPROVED** [review]

File: src/components/plots/PlotHeader.jsx [hunk 1]
```diff
@@ -3,27 +3,21 @@ import React, {
...
 import { getFromApiExpectOK } from '../../utils/getDataExpectOK';
 import { LOAD_CONFIG } from '../../redux/actionTypes/componentConfig';
 import { initialPlotConfigStates } from '../../redux/reducers/componentConfig/initialState';
 
-const Header = (props) => {
-  const {
-    experimentId, plotUuid, finalRoute,
-  } = props;
-
+const PlotHeader = ({ title, experimentId, plotUuid }) => {
```
Comment (aerlaut) [inline src/components/plots/PlotHeader.jsx:17]: Do you recommend we start using this new format to restructure props?

File: src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx [hunk 2]
```diff
@@ -337,35 +333,37 @@ const DotPlotPage = (props) => {
...
-              config={config}
-              onUpdate={updatePlotWithChanges}
-              renderExtraPanels={renderExtraPanels}
-              defaultActiveKey='gene-selection'
-            />
-          </Space>
-        </Col>
-      </Row>
-    </div>
+      <Space direction='vertical' style={{ width: '100%', padding: '0 10px' }}>
```
Comment (aerlaut) [inline src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx:342]: Having `Space` with only one child is not useful. `Space` is there to provide spacing between child components. If you only want, it's better to use `div`.

File: src/components/plots/PlotHeader.jsx [hunk 3]
```diff
@@ -154,37 +131,28 @@ const Header = (props) => {
...
-              </Button>
-            </Space>
-          )}
-        />
-      </Col>
-    </Row>
+    <Header
+      title={title}
+      extra={(
+        <Space>
```
Comment (aerlaut) [inline src/components/plots/PlotHeader.jsx]: There is no need to use `Space` here if it only contains 1 child component.

File: src/components/plots/PlotHeader.jsx (hunk 1 above)
Comment (kafkasl) [reply src/components/plots/PlotHeader.jsx:17]: Well not really, I just saw it somewhere and reused it. If there are not many properties looks good but I don't have a strong opinion.

File: src/components/plots/PlotHeader.jsx (hunk 3 above)
Comment (kafkasl) [reply src/components/plots/PlotHeader.jsx]: I removed it

**Comment by kafkasl** [comment]
> Good work with the plot header refactor! It makes things really clear. Btw, when I checked out the plot, the header contains duplicated buttons for feedback and invite. There should only be one. Can you please fix this?
> 
> image

fixed it

**Pull request merged**